src/torch/14-deeprank-db/deeprank-db.lua

```diff
@@ -15,49 +15,36 @@ local tiefvision_commons = require '0-tiefvision-commons/tiefvision_commons'
 local similarity_lib = require '14-deeprank-db/similarity_lib'
 local database = require('0-tiefvision-commons/tiefvision_config_loader').load().database
 
-function getInitialRefIndex(similarities)
-  local initIndex = 1
-  for refIndex = 1, similarities:size()[1] do
-    local allMinusOne = true
-    for testIndex = refIndex, similarities:size()[2] do
-      allMinusOne = allMinusOne and similarities[refIndex][testIndex] == -1.0
-    end
-    if (allMinusOne) then
-      return refIndex
-    end
-  end
-  return similarities:size()[1] + 1
-end
-
 function similarityDb()
   local dataFolder = tiefvision_commons.dataPath('db/similarity/img-similarity-deeprank')
   local testLines = tiefvision_commons.getFiles(dataFolder)
-  local similarities = torch.ones(#testLines, #testLines) * -1.0
-  -- local similarities = database.read('img-sup-similarity-db')
-  local initialReferenceIndex = getInitialRefIndex(similarities)
+  local similaritiesDb = 'img-sup-similarity-db'
```

Comment:
now that we are doing this for production, we could use better naming
`image-supervised-similarity-database`